code-style/scss/README.md

```diff
@@ -1,52 +1,257 @@
+    // styles for .block__element
+  }
+
+  &__button {
+    // styles for .block__element
+
+    &--dark {
+      // styles for .block__element--modifier
+    }
+  }
+}
+```
+
+---
+
+## ITCSS
+
+ITCSS stands for Inverted Triangle CSS. It separates CSS into sections to help developers manage CSS specificity and the cascade. There are seven layers of the inverted triangle. The idea is that you start with a wide reach and low specificity at the top, and as you go down, the reach narrows and the properties get more specific. This helps you avoid specificity issues.
+
+#### Settings
+
+When using a preprocessor, settings are used to define variables. This puts variables at the top of the cascade so they are accessible to all partials further down. No CSS selectors should be generated by the preprocessor from partials in this section.
+
+When not using a preprocessor, the settings area is a good place to write comments describing the colors and font stacks as a guide. If using features such as custom properties, those that are global custom properties should be described in this section.
+
+#### Tools
+
+This section is specifically for preprocessor global mixins and functions. No CSS should be generated by the preprocessor from partials in this section. Some of our favorite global mixins to use are [unbuttonize](https://gist.github.com/zastrow/bb43d2b256d52738e0ffdee7f825e79f) to remove default button styles and [delist](https://gist.github.com/zastrow/94d97dfe2e7e2808bcf4dc45268e6338) to remove default list styles from `<ol>` and `<ul>`.
+
+#### Generic
+
+Here we define any generic styles that are not specific to the styles of the site. This section should include any Reset or Normalize libraries and any preferential base styles for elements. There shouldn’t be any classes or ids used in this section.
+
+#### Elements
+
+This is where we define styling for bare HTML elements (like H1, A, etc.). These elements come with default styling from the browser, so we can redefine them here.
+
+#### Objects
+
+Objects are class-based selectors that define undecorated design patterns, for example [the media object](http://www.stubbornella.org/content/2010/06/25/the-media-object-saves-hundreds-of-lines-of-code/) (a pattern with an image on the left and descriptive content on the right, like a Tweet for example) from OOCSS.
+
+#### Components
+
+This section is where we define our specific UI components. This is where the majority of our work takes place, and our UI components are often composed of Objects and Components.
+
+#### Utilities
+
+Utilities are helper classes with the ability to override anything that goes before it in the triangle. Utility classes are very specific. Sometimes we use utility classes to set up spacing systems, which would consist of creating classes for certain sizes of padding or margin to be used anywhere on the site, or to establish a hide helper class that our JavaScript can interact with to toggle the visibility of elements.
```

Comment:
> Utilities, sometimes called "trumps",

Reply:
LOVE the example.

Reply:
Maybe note that ... "because of the overriding nature of utility classes, we sometimes choose to prefix the class name with `u-` for larger projects" 

That english is bad but I hope you understand.

Reply:
Haha, yes, makes sense